user_management/api/tests/test_views.py

```diff
@@ -1010,3 +1010,21 @@ def test_send_email(self):
 
         expected = 'example.com account validate'
         self.assertEqual(email.subject, expected)
+
+    def test_send_email_authenticated(self):
+        """Assert authenticated user can receive a new confirmation email."""
+        user = UserFactory.create()
+        data = {'email': user.email}
+        request = self.create_request('post', user=user, data=data)
```

Comment:
What happens when a user posts someone elses email address? Should we be doing something special?